notebooks/notebook_template.ipynb

```diff
@@ -31,6 +31,8 @@
       "source": [
```

Comment:
<p>It looks like the new line restarted numbering at 1 after bullet #3?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1204/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>